src/tests/PostTest.test.ts

```diff
@@ -290,28 +290,28 @@ describe('post tests', () => {
     const postController = ControllerFactory.post(conn);
     const userController = ControllerFactory.user(conn);
     const post = PostFactory.fakeTemplate();
-    post.user = UserFactory.fakeTemplate();
-    post.user.saved = [];
+    const user = UserFactory.fakeTemplate();
+    user.saved = []
+    post.user = user;
 
     await new DataFactory()
       .createPosts(post)
       .createUsers(post.user)
       .write();
 
-    let userResponse = (await userController.getUserByGoogleId('shungoGoogleID')).user;
-    expect(userResponse).not.toBeUndefined();
-    expect(userResponse?.saved).toEqual([]);
+    let postsResponse = (await postController.getSavedPostsByUserId(user));
```

Comment:
some of these lines no longer need to be wrapped in parentheses since they no longer access the user property